ws-aspnet/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM cloud9/workspace
+MAINTAINER Peter Bozovic <peter.bozovic@ogilvy.com>
+
+# Getting CoreClr dependencies
+RUN apt-get -y install libunwind8 gettext libssl-dev libcurl3-dev zlib1g libicu-dev
+
+# Installing Kestrel (web server)
+RUN LIBUV_VERSION=1.8.0 \
+    && sudo apt-get install make automake libtool curl \
+	&& sudo curl -sSL https://github.com/libuv/libuv/archive/v${LIBUV_VERSION}.tar.gz | sudo tar zxfv - -C /usr/local/src \
+    && cd /usr/local/src/libuv-$LIBUV_VERSION \
+	&& sudo sh autogen.sh \
+	&& sudo ./configure \
+	&& sudo make \
+	&& sudo make install \
+    && sudo rm -rf /usr/local/src/libuv-$LIBUV_VERSION \
+	&& cd ~/ \
+	&& sudo ldconfig
+	
+# Installing DNVM
+RUN sudo -u ubuntu -i bash -l -c "curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_BRANCH=dev sh"
+
+# Installing Yeoman to hande project templates
+RUN sudo -u ubuntu -i bash -l -c "sudo npm install -g npm"
+RUN sudo -u ubuntu -i bash -l -c "npm install -g yo generator-aspnet"
```

Comment:
This actually causes a permissions error during build for me. I believe it's because of using sudo on the step above, some files in ~/.nvm are no longer owned by ubuntu after that